streaming/streaming.go

```diff
@@ -4,40 +4,29 @@ import (
 	"fmt"
 
 	"github.com/aws/aws-sdk-go/aws"
-	"github.com/aws/aws-sdk-go/aws/session"
 	"github.com/aws/aws-sdk-go/service/kinesis"
-	"github.com/sul-dlss-labs/taco/config"
 )
 
 // Stream the interface for streaming pipeline
 type Stream interface {
 	SendMessage(string) error
 }
 
-type kinesisStream struct {
-	streamName *string
-	connection *kinesis.Kinesis
+type KinesisStream struct {
+	StreamName string
```

Comment:
Why have these been changed to be public? Are they used outside this package?

Reply:
I believe I was getting an error without them being public, but I'll test.

Reply:
Yes, they are set in main.go now in order to be passed though to the API/handler.